security-analysis/security-analysis-api/src/main/java/com/powsybl/security/json/SecurityAnalysisParametersSerializer.java

```diff
@@ -30,12 +30,21 @@ public void serialize(SecurityAnalysisParameters parameters, JsonGenerator jsonG
         jsonGenerator.writeStartObject();
 
         jsonGenerator.writeStringField("version", SecurityAnalysisParameters.VERSION);
+        jsonGenerator.writeFieldName("increased-violations-parameters");
+        jsonGenerator.writeStartObject();
+        jsonGenerator.writeNumberField("increased-flow-violations-proportional-threshold", parameters.getIncreasedViolationsParameters().getFlowProportionalThreshold());
+        jsonGenerator.writeNumberField("increased-low-voltage-violations-proportional-threshold", parameters.getIncreasedViolationsParameters().getLowVoltageProportionalThreshold());
+        jsonGenerator.writeNumberField("increased-high-voltage-violations-proportional-threshold", parameters.getIncreasedViolationsParameters().getHighVoltageProportionalThreshold());
+        jsonGenerator.writeNumberField("increased-low-voltage-violations-absolute-threshold", parameters.getIncreasedViolationsParameters().getLowVoltageAbsoluteThreshold());
+        jsonGenerator.writeNumberField("increased-high-voltage-violations-absolute-threshold", parameters.getIncreasedViolationsParameters().getHighVoltageAbsoluteThreshold());
+        jsonGenerator.writeEndObject();
 
         jsonGenerator.writeFieldName("load-flow-parameters");
         JsonLoadFlowParameters.serialize(parameters.getLoadFlowParameters(), jsonGenerator, serializerProvider);
 
         JsonUtil.writeExtensions(parameters, jsonGenerator, serializerProvider, JsonSecurityAnalysisParameters.getExtensionSerializers());
 
         jsonGenerator.writeEndObject();
+        jsonGenerator.writeRaw('\n');
```

Comment:
we should not add this, the exact formatting should be left to more general options (pretty printing etc)